api/src/main/java/org/pmiops/workbench/reporting/ReportingServiceImpl.java

```diff
@@ -61,7 +62,8 @@ public void collectRecordsAndUpload() {
     if (batchUploadSuccess) {
       reportingUploadService.uploadVerifiedSnapshot(captureTimestamp);
     } else {
-      logger.warning("Failed to verify batch upload result");
+      logger.severe("Failed to verify batch upload result");
+      throw new ServerErrorException("Failed to verify batch upload result");
```

Comment:
After catching the individual-table exceptions in https://github.com/all-of-us/workbench/pull/9109, an *overall* verification failure returned a positive response.  Now it will correctly return 500.